### ISSUE TITLE
Why BatchNormalization moving averages are set to be "trainable"?

### ISSUE THREAD

**Issue by TianwenWei** [davidsandberg/facenet#424]
Hello @davidsandberg , thank you very much for sharing your code. 
Here I have a question regarding the batch normalization parameters in the inception-resnet-v1 model that you used:

> batch_norm_params = {
>         # Decay for the moving averages.
>         'decay': 0.995,
>         # epsilon to prevent 0s in variance.
>         'epsilon': 0.001,
>         # force in-place updates of mean and variance estimates
>         'updates_collections': None,
>         # Moving averages ends up in the trainable variables collection
>         **'variables_collections': [ tf.GraphKeys.TRAINABLE_VARIABLES ],**
>     }

I don't get the point. Why you set deliberately variables_collections=tf.GraphKeys.TRAINABLE_VARIABLES ]? Shouldn't this parameter be left as its default value?

**Comment by davidsandberg** [comment]
To reduce the size of the model only trainable variables are stored. And this makes the batch norm variables included in the model.
